Bits/Bits.reds

```diff
@@ -0,0 +1,50 @@
+// -----------------------------------------------------------------------------
+// Codeware
+// -----------------------------------------------------------------------------
+//
+// Allows for storing up to 32 boolean-like flags packed in a single integer
+//
+
+module Codeware
+
+public class Bits {
+  public static func ShiftRight(num: Int32, n: Int32) -> Int32 {
+    return num / Bits.PowI(2, n);
+  }
+
+  public static func ShiftLeft(num: Int32, n: Int32) -> Int32 {
+    return num * Bits.PowI(2, n);
+  }
+
+  public static func PowI(num: Int32, times: Int32) -> Int32 {
+    return RoundMath(PowF(Cast<Float>(num), Cast<Float>(times)));
+  }
+
+  public static func Invert(num: Int32) -> Int32 {
+    let after = num;
+    let i: Int32 = 0;
+    while i < 32 {
```

Comment:
I might have messed up here, I need to test first.
Is there any way to display raw bits in .reds ?

Reply:
resolved on DM in discord, and thanks :)